_data/committers.yml

```diff
@@ -66,19 +74,31 @@
 - name: Dewey Dunnington
   role: PMC
   alias: paleolimbot
-  affiliation: Voltron Data
+  affiliation: Wherobots 
 - name: François Saint-Jacques
   role: PMC
   alias: fsaintjacques
   affiliation: Optable
+- name: Gang Wu
+  role: PMC
+  alias: gangwu
+  affiliation: Unknown
```

Comment:
CC @wgtmac but I assume you'd prefer a different alias and affiliation?

Reply:
I think that this `alias` is correct: https://people.apache.org/phonebook.html?uid=gangwu

We use Apache ID for `alias`.